src/openforms/submissions/models/submission_files.py

```diff
@@ -48,6 +49,20 @@ def select_prune(self, age: timedelta):
 
 class TemporaryFileUpload(DeleteFileFieldFilesMixin, models.Model):
     uuid = models.UUIDField(_("UUID"), unique=True, default=uuid.uuid4)
+    legacy = models.BooleanField(
+        _("legacy"),
+        default=False,
+        help_text=_("Whether the instance is linked to a submission instance."),
+    )
+    # TODO null=True is a transitional state, and should be removed
+    # at some point:
```

Comment:
put a `DeprecationWarning` in the comment so we can find it when preparing Open Forms 3.0